### ISSUE TITLE
[Keyboard Navigation – Make Code - Create a project] Keyboard focus indicator is not visible on 'Create button' under 'Create a project' pop-up.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3559]
**User Experience**:
If the desired component, on which keyboard operations will interact at any point in time is not visually determined, people with attention limitations, short term memory limitations, or limitations in executive processes will find it difficult to discover where the focus is located.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate to the controls appearing under "My Project" section in Home page and select 'New Project' Button.
4. Navigate all controls under 'Create a Project' pop-up.
5. Verify that keyboard focus indicator is visible on 'Create button' or not.

**Actual Result:**
Keyboard focus indicator is not visible on 'Create button' under 'Create a project' pop-up and due to this, visual impaired user is unable to see the keyboard focus on that control and also he/she is unable to perform functionality over such control.

**Expected Result:**
Keyboard focus indicator should be visible on 'Create button' under 'Create a project' pop-up, so that visual impaired user could easily see the keyboard focus on that control to perform functionality over such control.

**Note:**
Same issue appear under Chrome & Firefox.

**WCAG Reference:**
[WCAG 2.4.7 - Focus Visible
](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html})

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG2.4.7, #Win10-Edge(Chromium)

[WCAG2.4.7_Create a project_Focus indicator.zip](https://github.com/microsoft/pxt-microbit/files/5479668/WCAG2.4.7_Create.a.project_Focus.indicator.zip)

**Comment by jwunderl** [comment]
By default the buttons darken for selection per sui styling, in high contrast mode they display focus outline.

![image](https://user-images.githubusercontent.com/5615930/113220222-abe99280-9237-11eb-896a-d01a4afb6732.png)
